cob_bringup/robots/cob4-2.xml

```diff
@@ -93,7 +93,7 @@
 		</include>
 
 		<!-- startup hardware -->
-		<!--include file="$(find cob_bringup)/drivers/canopen_402.launch" >
+		<include file="$(find cob_bringup)/drivers/canopen_402.launch" >
 			<arg name="robot" value="$(arg robot)" />
 			<arg name="component_name" value="torso" />
 			<arg name="can_device" value="can1" />
```

Comment:
Torso is on can0 and sensorring on can1. Seems as either a merge went wrong or not everything from the robot is committed and pushed.
@ipa-nhg please check